src/main/java/com/stripe/net/ApiResource.java

```diff
@@ -85,6 +85,8 @@ private static String className(Class<?> clazz) {
       className = "issuing/" + className;
     } else if (clazz.getName().contains("com.stripe.model.sigma.")) {
       className = "sigma/" + className;
+    } else if (clazz.getName().contains("com.stripe.model.reporting.")) {
+      className = "reporting/" + className;
```

Comment:
I know we need to fix this but I think this PR is the wrong place. Though we should schedule this work soon (happy to take, we just need to sync on it a bit)

Reply:
👍